docs/design/rosbag2_playback_time.md

```diff
@@ -187,6 +187,53 @@ No special allowance needs to be made in the `Player` - time will be provided at
 
 The `Player` must register a `JumpHandler` with the `Clock` - so that when a jump occurs, the current message playback queue can be invalidated and re-enqueued according to the new starting time.
 
+## Time Synchronization and Conversion
+
+When implementing this time control, there are two separate time streams to consider:
+1. "ROS time": this is the time that is related to the messages in the bag - this time may speed up, slow down, pause, or jump based on rosbag playback controls (or external Time Sources)
+2. "Steady time": this time always moves forward, and always moves at the same rate; it is the real time that the user watching a playback experiences
```

Comment:
There is no third timeline. The "original timestamps in messages" _is_ the ROS Time that I'm talking about. It is _also_ the ROS Time at playback. They are the same thing. Rosbag2 says "it is time 2", when it needs to play "message with timestamp = 2" - these match up exactly. There is no difference between "timestamp when you are recording" and "timestamp when you are playing". The only difference is the frame of reference in which you are standing. You can swap the two timelines and re-scale so that one is constant and the other looks distorted, but this is a simple transformation between coordinate spaces, this is not a "new thing"

> If I understand correctly "Steady time" - this is our "watch clock" time or OS time in which we call std::chrono::sleep_until(..) right?

Yes - it's generally queried with `std::chrono::steady_clock::now`